openshift/machine-config-operator
# Bug 1875181: [4.4] crio: bump to v1.17.5

**- What I did**
when the crio version falls out of sync with the verison in openshift, we end up rendering the incorect ignition config

This fixes a problem where, upon creating a ctrcfg, we lose our default_env value

Signed-off-by: Peter Hunt <pehunt@redhat.com>

**- How to verify it**
run `crio config` on a node
apply any ctrcfg
run `crio config` again
check the diff, noting the changes didn't strictly include that which came in the ctrcfg

note: this would be a good e2e test to add, however, running `crio config` on a node has proven to be challenging. One has to chroot, or else there will be ldd issues. However, a combination of the way the go os/exec package works, along with the need to run `chroot/ host /bin/bash -c 'crio config` has caused it to just fail for me. not sure why yet.
**- Description for the changelog**
